tests/eas/acceptance.py

```diff
@@ -28,15 +28,15 @@
 from env import TestEnv
 from test import LisaTest, experiment_test
 
-# Read the config file and update the globals
```

Comment:
I think this is Github being silly, my git log looks like this:

```
0600010 * lisa-test-interface-change tests/eas/acceptance: WakeMigration: Use Python dict not JSON
688e012 * tests/eas/acceptance: Offload+IdlePull: Use Python dict not JSON
1fa29f4 * tests/eas/acceptance: SmallTaskPacking: Use Python dict not JSON
675b792 * tests/eas/acceptance: ForkMigration: use Python dict not JSON
3320677 * tests/eas/acceptance: Remove JSON config
ea0b921 * LisaTest: Flesh out docstrings
6fd0f93 * LisaTest: Add methods to override for test configuration
4d62d29 * ipynb/tutorial: Update Executor tutorial notebook
051a6e6 * ipynb/utils: Update Executor example notebook
1a44422 * executor: Extend docstring
394991f * executor: Rename tests_conf to experiments_conf
acd8c57 * executor: Take a TestEnv instead of a target_conf
```